onshape_to_robot/load_robot.py

```diff
@@ -179,7 +180,7 @@ def assignParts(root, parent):
                 print(Fore.RED + 'ERROR: "' + name +
                       '" is declared as a DOF but the mate type is '+data['mateType']+'')
                 print(
-                    '       Only REVOLUTE, CYLINDRICAL, SLIDER and FASTENED are supported' + Style.RESET_ALL)
+                    '       Only REVOLUTE, CYLINDRICAL, SLIDER, CONTINUOUS and FASTENED are supported' + Style.RESET_ALL)
```

Comment:
This warning refers to OnShape mates that are supported, and there is no such mate as CONTINUOUS

Reply:
😬 Ops.. Warning statement is corrected.